.changeset/dry-eagles-sleep.md

```diff
@@ -0,0 +1,6 @@
+---
+"@getflip/swirl-icons": minor
+"@getflip/swirl-components": patch
```

Comment:
Issue: "@getflip/swirl-components-angular" and "@getflip/swirl-components-react" are missing.